documentation/extraction-process.md

```
@@ -1,45 +1,45 @@
+- edit: `$extraction-framework/dump/extraction.spark.properties`
+- `../run sparkextraction extraction.spark.properties`
+
+## 3. Mappings-Extraction 
+The Mappings-Extraction produces better data than the generic-spark extraction using community-made mapping-files. Due to the complexity of this task, the mapping-extraction is currently run with the non-Apache Spark version of the extraction-framework:
+- `cd $extraction-framework/dump`
+- edit: `$extraction-framework/dump/extraction.mapping.properties`
+- `../run extraction extraction.mapping.properties`
+
+## 4. Wikidata-Extraction
+### 4.1 Extract the Data
+ - `cd $extraction-framework/dump`
```

markdownlint-cli2 (0.18.1)

Unordered list indentation
Expected: 0; Actual: 1

(MD007, ul-indent)